app/styles/live.module.css

```diff
@@ -42,7 +43,7 @@
 	margin-top: 1rem;
 }
 
-@media only screen and (max-width: 768px) {
+@media only screen and (max-width: 1000px) {
```

Comment:
Please change the value of the media query according to the standards
769px — 1024px: Small screens, laptops
1025px — 1200px: Desktops, large screens
1201px and more —  Extra large screens, TV